### DISCUSSION TITLE
Usage example with other simulators

### DISCUSSION THREAD

**Discussion by scotttravis1** [NVlabs/curobo#156] [Software Q&A]
Thanks for the amazing work!

I was wondering if it's possible to integrate curobot with other simulators like (MuJoCo etc), specifically for motion planning. If yes, are there examples available for the same?

**Comment by balakumar-s** [comment]
cuRobo is a python library that can be integrated with any simulator. We only provide Isaac Sim examples at the moment. Also relevant: https://github.com/NVlabs/curobo/discussions/8#discussioncomment-7440177
